controls/views.py

```diff
@@ -738,6 +738,16 @@ def statement_id_from_control(control_id, part_id):
 
         return f"{control_id}"
 
+    def generate_source(self, src_str):
+        """Return a valid catalog source given string"""
+        DEFAULT_SOURCE = "NIST_SP-800-53_rev5"
+        if src_str is None or src_str=='':
```

Comment:
if not src_str:
            return DEFAULT_SOURCE

Reply:
fixed.